Tools/autotest/arducopter.py

```diff
@@ -9269,6 +9269,83 @@ def EKFSource(self):
         if ex is not None:
             raise ex
 
+    def test_EK3_ext_nav_vel_without_vert(self):
+        """
+        Test that EK3 External Navigation velocity works without vertical velocity.
+        """
```

Comment:
Usually we we'd have
```suggestion
        """Test that EK3 External Navigation velocity works without vertical velocity."""
```